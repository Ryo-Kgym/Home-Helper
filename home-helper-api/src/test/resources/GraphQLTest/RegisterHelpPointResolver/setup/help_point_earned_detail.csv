id,achievement_id,item_id
1,1,aaa
2,1,bbb