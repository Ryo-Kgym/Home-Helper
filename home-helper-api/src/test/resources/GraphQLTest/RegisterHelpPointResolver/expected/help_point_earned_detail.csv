id,achievement_id,item_id
1,1,aaa
2,1,bbb
3,2,aaa
4,2,bbb
5,2,ccc